For Advanced/070.md

```diff
@@ -0,0 +1,32 @@
+# Chasing Dots
+
+Time for some fun! Let’s play a game — or rather let robot play it. Run this script to see the robot’s flawless victory in mimicking mouse clicks on red dots (be sure to enable the Images Addon before you press **F9** key):
+
+```G1ANT
+♥image = ♥environment⟦USERPROFILE⟧\Desktop\dot.png
+file.download https://github.com/G1ANT-Robot/G1ANT.Manual/blob/develop/-assets/dot.png filename ♥image
+program chrome arguments https://mouseaccuracy.com/game
+window ✱Accuracy✱ style maximize
+
+while ⊂true⊃
+  label retry
+  image.find image1 ♥image timeout 2 errorjump retry
+  mouse.click ♥result mousedelay 2
+end
+```
+
+What happened here? First, the robot downloaded a tiny image (3x3px) from our GitHub repository to a user’s Desktop. Then it opened Chrome (or other browser of your choice, if you’d provided its name in the third line) and navigated to the Mouse Accuracy game page. The browser window was maximized and the game started: a user is supposed to click the red dots that appear on the screen.
+
+The whole robot magic happened within the `while` loop. Normally, this loop is executed as long as the condition given as the `while` command argument is true. Here, this condition is always `⊂true⊃`, so the loop is infinite.
+
```

Comment:
You should also notify user to use ctrl+f12 to stop infinite script here i think

Reply:
yep, you're right